[Orc8r] 5G WS#2-MVC1 User Guide

GANESH-WAVELABS opened the pull request:
Signed-off-by: GANESH IRRINKI <ganesh.irrinki@wavelabs.ai>

<!--
    OCN 5G WS#2 - Orchestration MVC1(Monitoring stack for 5G Core Containers)
Kibana Dashboard configured to view AGW Syslogs.
Script to deploy K8s Cluster, Monitoring Stack, AGW VM (Magma Dev)
User Guide/README document for deployment steps - K8s Cluster, Monitoring Stack, AGW VM (Magma Dev)
-->

## Summary

OCN 5G WS#2 - Orchestration MVC1(Monitoring stack for 5G Core Containers)
Kibana Dashboard configured to view AGW Syslogs.
Script to deploy K8s Cluster, Monitoring Stack, AGW VM (Magma Dev)
User Guide/README document for deployment steps - K8s Cluster, Monitoring Stack, AGW VM (Magma Dev)

## Test Plan

<!--
    How did you test your change? How do you know it works?
    Add supporting screenshots, terminal pastes, etc. as necessary
-->

## Additional Information

- [ ] This change is backwards-breaking

<!--
    If this is a backwards-breaking change, document the upgrade instructions.
    All upgrade instructions for backwards-breaking changes will be aggregated
    in the next release's changelog so this is very important to fill out.
-->

GANESH-WAVELABS committed:
[Orc8r] 5G WS#2-MVC1 User Guide

Signed-off-by: GANESH IRRINKI <ganesh.irrinki@wavelabs.ai>

mpgermano commented on cn/deploy/scripts/MVC1_5G_Orc8r_deployment_script.py:
```diff
@@ -0,0 +1,379 @@
+#Dictionary to maintain k8s services whether they are Running or not
+k8s_obj_dict = {}
+#Build context for Docker files for AMF, SMF, UPF
+BUILD_CONTEXT_PARENT_DIR = '/tmp/Orc8r_docker'
```
nit: lets name this `/tmp/orc8r_docker`

mpgermano commented on cn/deploy/scripts/MVC1_5G_Orc8r_deployment_script.py:
```diff
@@ -0,0 +1,379 @@
+#!/usr/bin/env python3
```
Ni: can we rename this file `mvc1_5g_orc8r_deployment_script.py`

mpgermano reviewed (changes requested):
This looks good. Can you please address the static analysis issues. Then I can approve.

GANESH-WAVELABS commented:
Hi Michael,

Thanks for the review for the PR(#4515), I have modified the code according to your two comments/suggestions.

1)  renamed the file  "MVC1_5G_Orc8r_deployment_script.py" to "mvc1_5g_orc8r_deployment_script.py"
2) renamed the file '/tmp/Orc8r_docker' to '/tmp/orc8r_docker'

Also fixed all the possible staic analysis issues. Could you please review it and help me to approve & merge this PR.

Thanks & Regards,
Ganesh . Irrinki,
Email-Id : ***@***.*** ,
Mobile : +91 8121666196, +91 8867839739,
Wavelabs Technologies PVT LTD
________________________________
From: Michael Germano ***@***.***>
Sent: Friday, March 26, 2021 8:26 AM
To: magma/magma ***@***.***>
Cc: Ganesh Irrinki ***@***.***>; Author ***@***.***>
Subject: Re: [magma/magma] [Orc8r] 5G WS#2-MVC1 User Guide (#4515)


CAUTION: This email originated from outside of the organization. Do not click links or open attachments unless you recognize the sender and know the content is safe.


@mpgermano requested changes on this pull request.

This looks good. Can you please address the static analysis issues. Then I can approve.

________________________________

In cn/deploy/scripts/MVC1_5G_Orc8r_deployment_script.py<https://github.com/magma/magma/pull/4515#discussion_r601979887>:

> +limitations under the License.
+"""
+
+import sys, getopt, shutil
+import time, re, logging
+import os, subprocess
+import webbrowser, socket
+import threading, platform
+
+
+#Initialize the lock
+lock = threading.Lock()
+#Dictionary to maintain k8s services whether they are Running or not
+k8s_obj_dict = {}
+#Build context for Docker files for AMF, SMF, UPF
+BUILD_CONTEXT_PARENT_DIR = '/tmp/Orc8r_docker'


nit: lets name this /tmp/orc8r_docker

________________________________

In cn/deploy/scripts/MVC1_5G_Orc8r_deployment_script.py<https://github.com/magma/magma/pull/4515#discussion_r601982343>:

> @@ -0,0 +1,379 @@
+#!/usr/bin/env python3


Ni: can we rename this file mvc1_5g_orc8r_deployment_script.py

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/magma/magma/pull/4515#pullrequestreview-621744174>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AQ45OPESVMGYTDILVUTA6YDTFPZWLANCNFSM4WO5VGNQ>.

mpgermano reviewed (approved):
Looks good. Thanks for this.

electronjoe commented:
There are two new tests in the CI pipeline (well not new tests, but newly running in CI and newly blocking merges).  Can you rebase and push so that CI runs?  The tests are the following.

```
ci/circleci: mme_test
ci/circleci: session_manager_test
```

[Edit] These tests will not actually have to run because no c/cpp is touched, but Github requires them to be evaluated as un-necessary for the merge to occur, which requires the CircleCI test be defined in your branch - which requires the rebase.
